index.html

```diff
@@ -321,9 +281,8 @@ <h3>CONTACT</h3>
     </main>
  <!--Footer section-->   
     <footer class="footer--section">
-        <hr>
-        <div class="supporting-item-container">    
-            <ul class="social--icons--container bullet--icons">
+        <div>    
+            <ul class="social--icons--container bullet--icons icons--footer">
```

Comment:
- [ ] Looks like changes broke the mobile version of the footer. You should not break the mobile design while implementing your desktop version. Email should be below social icons in the mobile version.
 
![Screenshot 2021-06-11 at 02-14-47 Alef's Portfolio](https://user-images.githubusercontent.com/1642014/121608623-0b93c380-ca5b-11eb-80a9-0f816aa133ac.png)